### ISSUE TITLE
Having problems running a custom score query

### ISSUE THREAD

**Issue by RoyLee** [karmi/retire#635]
I would like to elevate certain keywords that I've stored in an array (https://github.com/elasticsearch/elasticsearch/issues/1066) but am getting the following errors:

``` ruby
search_query = 'maz'
sponsored_brands = ['ford', 'jeep', 'mazda']
search = Tire.search 'cars' do
  query do
    custom_filters_score do
      query { term :brand,  search_query }
        filter do
          filter :term, :brand => sponsored_brands,
          script '_score * 2.0' # Getting an Error here
        end
      end
    end
  end
end
```

Error:

``` ruby
SyntaxError (/Users/proj/test_proj/app/controllers/cars_controller.rb:29: syntax error, unexpected tSTRING_BEG, expecting keyword_do or '{' or '('
            script '_score * 2.0'
                    ^):
```

I've searched various postings linked below but am not able to get the syntax to work correctly.
https://github.com/karmi/tire/commit/1b9b157ed5765b4cbd7bddf49c8edb18f67438a6
https://raw.github.com/karmi/tire-contrib/360348a12f2c2cd49f10b078a9595bd20c07dbdc/lib/tire/queries/custom_filters_score.rb

**Comment by karmi** [comment]
The documentation is incorrect in the contributed module, see test for proper syntax: https://github.com/karmi/tire-contrib/blob/master/test/queries/custom_filters_score/custom_filters_score_test.rb

@presdo: Can you look at the docs, please?

**Comment by presdo** [comment]
Karmi,

I m not sure what do you want me to look to? Which file does not have the right documentation?

Thanks.
- Jerry

**Comment by RoyLee** [comment]
I looked at the test example and tried the following:

``` ruby
search_query = 'maz'
sponsored_brands = ['ford', 'jeep', 'mazda']
search = Tire.search 'cars' do
  query = Query.new.custom_filters_score do
      query { term :brand, 'bmw' }
      filter do
        filter :terms, :tags => sponsored_brands
        boost 2.0
      end
      filter do
        filter :terms, :tags => sponsored_brands
        script '_score * 2.0'
      end
    end
  end
end
```

I am getting the following error:

``` ruby
NameError (uninitialized constant SearchesController::Query):
```

**Comment by karmi** [comment]
@presdo, https://github.com/karmi/tire-contrib/blob/master/lib/tire/queries/custom_filters_score.rb#L31-L32

**Comment by karmi** [comment]
@RoyLee The `query = Query. (...)` part is not correct. Just use the query type normally:

``` ruby
search_query = 'maz'
sponsored_brands = ['ford', 'jeep', 'mazda']

search = Tire.search 'cars' do
  query do
    custom_filters_score do

      query { term :brand, 'bmw' }
      filter do
        filter :terms, :tags => sponsored_brands
        boost 2.0
      end
      filter do
        filter :terms, :tags => sponsored_brands
        script '_score * 2.0'
      end

    end
  end
end
```

@presdo Can you add an example with the DSL integration into tests?

**Comment by RoyLee** [comment]
Thanks for looking into this.  I don't know exactly what is going on but I ran the exact code and am now getting the following error:

``` ruby
NoMethodError (undefined method `custom_filters_score' for #<Tire::Search::Query:0x007f92b1591e38 @value={}>):
```

Do I need to add a require somewhere to access that functionality? 
I am running Tire v0.5.4

**Comment by karmi** [comment]
> Do I need to add a require somewhere to access that functionality?

Indeed! [`require 'tire/queries/custom_filters_score'`](https://github.com/karmi/tire-contrib/blob/master/lib/tire/queries/custom_filters_score.rb) :)

**Comment by RoyLee** [comment]
I added the require line to the first line of the controller file where I'm running the search method and I get the following error:

``` ruby
LoadError (cannot load such file -- tire/queries/custom_filters_score):
app/controllers/searches_controller.rb:1:in `<top (required)>'
```

After researching this online, it looks like the solution for when developers encounter this issue is requires shouldn't be needed and simply bundle installing the gem should allow for access.  Is there something I'm missing here?  I also tried putting the require in my model with a method that has the same custom_filters_score code and it gave me the same error.
